doc/src/manual/arrays.md

```diff
@@ -872,10 +872,16 @@ slower than multiplication. While some arrays — like [`Array`](@ref) itself 
 are implemented using a linear chunk of memory and directly use a linear index
 in their implementations, other arrays — like [`Diagonal`](@ref) — need the
 full set of cartesian indices to do their lookup (see [`IndexStyle`](@ref) to
-introspect which is which). As such, when iterating over an entire array, it's
-much better to iterate over [`eachindex(A)`](@ref) instead of `1:length(A)`.
-Not only will the former be much faster in cases where `A` is `IndexCartesian`,
-but it will also support [OffsetArrays](https://github.com/JuliaArrays/OffsetArrays.jl), too.
+introspect which is which).
+
+!!! warnings
+
+    When iterating over the an entire array, and the values of the indices are needed, is
+    better to iterate over [`eachindex(A)`](@ref) instead of `1:length(A)`.
```

Comment:
I could be wrong since I'm not a native English speaker. My limited English grammar knowledge tells me that it should be:

```suggestion
    When iterating over the entire array with the values of the indices needed, it is
    better to iterate over [`eachindex(A)`](@ref) instead of `1:length(A)`.
```